assets/scss/nhsblocks/legacy/legacy.scss

```diff
@@ -0,0 +1,150 @@
+.nhsuk-panel-with-label {
+  width: 100%;
+
+  .nhsuk-panel-with-label__label {
+	a, a:visited {
+	  color: $color_nhsuk-white;
+	}
+  }
+
+  .entry-footer {
+	span {
+	  padding: 0 10px;
+	}
+  }
+}
+
+/*panel extension*/
+.nhsuk-panel.is-style-panel-grey {
+  @extend .nhsuk-panel--grey;
+}
+
+.nhsuk-panel.is-style-panel-with-label {
+  @extend .nhsuk-panel-with-label;
+
+  h3, h3 .rich-text {
+	@extend .nhsuk-panel-with-label__label;
+  }
+}
+
+/* promo extension */
+.nhsuk-promo.is-style-promo-small {
+  @extend .nhsuk-promo--small;
+}
+.nhsuk-promo {
+  figure {
+	margin: 0;
+  }
+}
+
+.wp-block-nhsblocks-panel1 {
+  .paneltext {
+	display: flow-root;
```

Comment:
flow-root does not get supported by IE11, shall we replace it with anything that actually works with IE